components/widgets/admin_console/admin_header.test.jsx

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+import sinon from 'sinon';
+
+import AdminHeader from './admin_header.jsx';
+
+describe('components/widgets/admin_console/AdminHeader', () => {
+    test('render component with child', () => {
+        const wrapper = shallow(<AdminHeader>{'Test'}</AdminHeader>);
+        expect(wrapper).toMatchInlineSnapshot(`
+<h3
+  className="admin-console-header"
+>
+  Test
+</h3>
+`
+        );
+    });
+
+    test('children prop is mandatory', () => {
+        const stub = sinon.stub(console, 'error');
+
+        shallow(<AdminHeader/>);
+
+        expect(stub.calledOnce).toEqual(true);
+        expect(stub.getCall(0).args[0]).toMatch(/Warning: Failed prop type: The prop `children` is marked as required in `AdminHeader`, but its value is `undefined`./);
+
+        console.error.restore();
+    });
```

Comment:
I think we can do the same using Jest, something like:
```javascript
    test('children prop is mandatory', () => {
        console.error = jest.fn();

        shallow(<AdminHeader/>);

        expect(console.error).toBeCalledTimes(1);
        expect(console.error).toBeCalledWith('Warning: Failed prop type: The prop `children` is marked as required in `AdminHeader`, but its value is `undefined`.\n    in AdminHeader');
    });
```

Reply:
Done.

Reply:
Done.